Tests/SonarQube.Bootstrapper.Tests/DefaultPropertiesFileTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DefaultPropertiesFileTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
```

Comment:
Added some simple checks for the default SonarQube.Analysis.xml i.e. that the XML is valid and that it contains the expected default values.